### PR TITLE
DM-3165: Adding changelogs for DM-3165 migration of springfox to spri…

### PR DESCRIPTION
**Pull request by shaikhahme**
…ngdoc y2024

### CODE REVIEW

File: content/change-logs/device-management/content/lwm2m-agent-10-18-540-195-Security-improvement-migration-springfox-to-springdoc.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+date:
+title: LWM2M service security improvement by replacing deprecated library
```
Comment (BeateRixen) [inline content/change-logs/device-management/content/lwm2m-agent-10-18-540-195-Security-improvement-migration-springfox-to-springdoc.md:3]:
```suggestion
title: Replaced deprecated library to improve LWM2M agent security
```

File: content/change-logs/device-management/content/lwm2m-agent-10-18-540-195-Security-improvement-migration-springfox-to-springdoc.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-ggH2M4hf3
+    label: lwm2m-agent
+ticket: DM-3165
+version: 10.18.540.195
+---
+
+To improve security, the deprecated Swagger/OpenAPI library used by the LWM2M service has been replaced by a new supported library.
```
Comment (BeateRixen) [inline content/change-logs/device-management/content/lwm2m-agent-10-18-540-195-Security-improvement-migration-springfox-to-springdoc.md:18]:
```suggestion
To improve security, the deprecated Swagger/OpenAPI library used by the LWM2M agent has been replaced by a new supported library.
```

File: content/change-logs/device-management/content/lwm2m-agent-10-18-540-195-Security-improvement-migration-springfox-to-springdoc.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+product_area: Device Management & Connectivity
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-ggH2M4hf3
+    label: lwm2m-agent
```
Comment (BeateRixen) [inline content/change-logs/device-management/content/lwm2m-agent-10-18-540-195-Security-improvement-migration-springfox-to-springdoc.md:13]: must be "cumulocity" for 2024

**Comment by BeateRixen** [comment]
@shaikhahme please make sure that you add the file at the right location. The related PR for develop created an extra folder "content" containing this one file within the folder "change logs/device-management" which should not be the case.

File: content/change-logs/device-management/content/lwm2m-agent-10-18-540-195-Security-improvement-migration-springfox-to-springdoc.md (hunk 3 above)
Comment (shaikhahme) [reply content/change-logs/device-management/content/lwm2m-agent-10-18-540-195-Security-improvement-migration-springfox-to-springdoc.md]: understood. resolved in dbe3f722af9283642cc3f7db66c4006ebfa73999

**Comment by shaikhahme** [comment]
> @shaikhahme please make sure that you add the file at the right location. The related PR for develop created an extra folder "content" containing this one file within the folder "change logs/device-management" which should not be the case.

Sorry for the error, resolved in dbe3f722af9283642cc3f7db66c4006ebfa73999

**Review by Oezge-Akin — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**